openshift/datasync-showcase-amq.yml

```diff
@@ -263,21 +227,17 @@ objects:
           volumeMounts:
           - mountPath: /usr/src/app/files
             name: files-storage
+          - mountPath: /website/assets/config
```

Comment:
the app isn't located at `/` inside the image. This path should be `/usr/src/app/website/assets/config`